certbot/certbot/util.py

```diff
@@ -563,11 +563,11 @@ def get_strict_version(normalized):
     :param str normalized: normalized version string
 
     :returns: An equivalent strict version
-    :rtype: distutils.version.StrictVersion
+    :rtype: distutils.version.LooseVersion
 
     """
     # strict version ending with "a" and a number designates a pre-release
-    return distutils.version.StrictVersion(normalized.replace(".dev", "a"))
+    return distutils.version.LooseVersion(normalized.replace(".dev", "a"))
```

Comment:
This function is part of Certbot public API and so its behavior should not change.

Reply:
Agreed, removed.